index.js

```diff
@@ -150,11 +150,15 @@ function mungeCmd (workingDir, options) {
 
   var m = command.match(re)
   var replace
+  var nodeCmd = workingDir + '/node.cmd'
```

Comment:
this should probably be `\\node.cmd`?

Reply:
Not necessarily. afaik, JS on windows can parse "/" paths properly. One can use e.g. the "path" package:

    var path = require("path");
    var absolutePath = path.resolve("Relative file path");